src/providers/trusted_service/context/ts_protobuf.rs

```diff
@@ -79,6 +81,7 @@ set_handle_impl!(VerifyHashIn);
 set_handle_impl!(AsymmetricEncryptIn);
 set_handle_impl!(AsymmetricDecryptIn);
 set_handle_impl!(ExportPublicKeyIn);
+set_handle_impl!(ExportKeyIn);
 
 impl Drop for ImportKeyIn {
```

Comment:
Could you please implement `Drop` for `ExportKeyOut`, scrubbing away the `data` field? We do this for all operations that handle sensitive data such as "raw" key material. (I'm checking now if we should actually do the same for `GenerateRandomOut`, since that output could be used for sensitive stuff)

Reply:
Did you decide if this should also be implemented for `GenerateRandomOut`?

Reply:
Ah, yes - yeah, could you add `GenerateRandomOut` there as well? The output isn't really "defined" as sensitive, but it could reasonably be used for quite sensitive stuff.